statline_bq/utils.py

```diff
@@ -951,6 +867,23 @@ def get_col_descs_from_gcs(
     return col_desc
 
 
+def remove_dir(dir):
```

Comment:
Not sure whether it is necessary to wrap standard utility functions like this. Feels unnecessary to me. If you don't like the name you could just do:

```
from shututil import rmtree as remove_dir
```